step.yml

```diff
@@ -130,6 +130,15 @@ inputs:
       - xcpretty
       - xcodebuild
       is_required: true
+  - export_uitest_artifacts: "false"
+    opts:
+      title: Export UITest Artifacts
+      description: |
+        If true, step will export the attachments of the UITest result into BITRISE_DEPLOY_DIR.
```

Comment:
If enabled, the attachments of the UITest will be exported into the `BITRISE_DEPLOY_DIR`, as a compressed ZIP file.
Attachments include screenshots taken during the UI test, and other artifacts.